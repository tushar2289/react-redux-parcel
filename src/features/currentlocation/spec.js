import currentLocationReducer, { setCurrentLocation } from './slice';

describe('Current location reducer', () => {
  let initialState;
  beforeEach(() => {
    initialState = currentLocationReducer(undefined, {});
  });

  it('should return initial state', () => {
    expect(initialState).toMatchSnapshot();
  });

  it('should set theme', () => {
    expect(
      currentLocationReducer(initialState, {
        type: setCurrentLocation.type,
        payload: {
          lat: 100,
          lng: 100,
        },
      })
    ).toMatchSnapshot();
  });
});
