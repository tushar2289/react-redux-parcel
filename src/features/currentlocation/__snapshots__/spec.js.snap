// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Current location reducer should return initial state 1`] = `
Object {
  "lat": 19.170078,
  "lng": 72.860186,
}
`;

exports[`Current location reducer should set theme 1`] = `
Object {
  "lat": 100,
  "lng": 100,
}
`;
